pvsystemprofiler/algorithms/performance_model_estimation.py

```diff
@@ -6,23 +6,54 @@
 """
 
 
-def find_fit_costheta(data_matrix, clear_index):
+def find_fit_costheta(data_matrix, clear_index, doy):
     """
     :param data_matrix: power matrix.
     :param clear_index: boolean array specifying clear days.
```

Comment:
We should add the doy param here. Is it also as below?
```suggestion
    :param doy: day of year (float or array)
```